Software/CSharp/GrovePi/DeviceFactory.cs

```diff
@@ -47,11 +48,11 @@ internal class DeviceBuilder : IBuildGroveDevices
     {
         private const string I2CName = "I2C1"; /* For Raspberry Pi 2, use I2C1 */
         private const byte GrovePiAddress = 0x04;
-        private const byte DisplayRgbI2CAddress = 0x62;
-        private const byte DisplayTextI2CAddress = 0x3e;
+        private const byte DisplayRgbI2CAddress = 0xC4;
```

Comment:
Firstly, i found that mini motor's address is not same with [Wiki](http://wiki.seeed.cc/Grove-Mini_I2C_Motor_Driver_v1.0/#change-default-i2c-address). 
Then i found them have one address is same. So i think change it should be clear. 
refer to: https://github.com/Seeed-Studio/Grove_LCD_RGB_Backlight/blob/master/rgb_lcd.h  

BTW, seeed will be change default minimotr driver's i2c address later.